src/components/UI/Cart/OrderSummary/Header/Header.jsx

```diff
@@ -0,0 +1,16 @@
+import React, { useContext } from "react";
+import { CartContext } from "../../../../../contexts/Cart/CartContext";
+
+const Header = () => {
+  const {
+    cartState: { cartItems },
+  } = useContext(CartContext);
```

Comment:
You are doing is great but you can use a custom hook for accessing the contexts.